modules/issue_tracker/php/uploadhelper.class.inc

```diff
@@ -156,10 +178,13 @@ class UploadHelper
             $DB->insert('issues_attachments', $values);
             // Move file from tmp to issue_tracker attachment directory.
             $path = $fileToUpload->full_path . '/' . $fileToUpload->file_hash;
-            move_uploaded_file($_FILES['file']['tmp_name'], $path);
+            move_uploaded_file($tmpFilePath, $path);
         } catch (\DatabaseException $e) {
             $this->endWithFailure();
-            error_log($e->getMessage());
+            $this->_logger->log(
```

Comment:
I think this is the wrong logger and there's no obvious way to fix it. It's using the Request logger instead of the Exception one since you're passing the one from the page class.

Maybe leave this an `error_log` for now (or remove the try/catch if that's an option.)

Reply:
actually you can just re-throw `$e` after the endWithFaliure call.

Reply:
@driusan updated